pulp_rpm/app/serializers/repository.py

```diff
@@ -210,12 +211,12 @@ class RpmPublicationSerializer(PublicationSerializer):
 
     metadata_checksum_type = serializers.ChoiceField(
         help_text=_("The checksum type for metadata."),
-        choices=CHECKSUM_CHOICES,
+        choices=ALLOWED_CHECKSUM_CHOICES,
```

Comment:
Perhaps we should allow it to be specified, but reject it in the viewset?  This would allow the error message to be more specific.

Reply:
the error message is something like: `"md5" is not a valid choice`

Reply:
Right, but "md5" is a valid choice in general, it's just not valid in this situation where the admin has not allowed it via configuration

Reply:
so I think we can close this PR, as this validation is already happening: https://github.com/pulp/pulp_rpm/blob/main/pulp_rpm/app/serializers/repository.py#L109-L117